### ISSUE TITLE
Docker stops firewalld starting properly on reboot

### ISSUE THREAD

**Issue by ajostergaard** [docker/docker#13433]
After a reboot have to login on the console as all ports are closed.

```
$ tail /var/log/firewalld
2015-05-24 07:44:46 ERROR: '/sbin/iptables -N FWDI_public -t filter' failed: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
2015-05-24 07:44:46 ERROR: COMMAND_FAILED: '/sbin/iptables -N FWDI_public -t filter' failed: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
2015-05-24 07:44:46 ERROR: '/sbin/iptables -N FWDI_public -t filter' failed: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
2015-05-24 07:44:46 ERROR: COMMAND_FAILED: '/sbin/iptables -N FWDI_public -t filter' failed: Another app is currently holding the xtables lock. Perhaps you want to use the -w option?
```

Starting firewalld manually enables remote access.

Docker doesn't seem to notice any issues:

```
# grep docker /var/log/messages | tail
May 24 07:44:46 dxbserver NetworkManager[676]: <info>  (docker0): device state change: secondaries -> activated (reason 'none') [90 100 0]
May 24 07:44:46 dxbserver NetworkManager[676]: <info>  (docker0): Activation: successful, device activated.
May 24 07:44:46 dxbserver nm-dispatcher: Dispatching action 'up' for docker0
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="-job init_networkdriver() = OK (0)"
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="Loading containers: start."
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="Loading containers: done."
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="docker daemon: 1.6.0 8aae715/1.6.0; execdriver: native-0.2; graphdriver: devicemapper"
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="+job acceptconnections()"
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="-job acceptconnections() = OK (0)"
May 24 07:44:47 dxbserver docker: time="2015-05-24T07:44:47+04:00" level=info msg="Daemon has completed initialization"
```

Disabling docker resolves the issue.

```
$ yum info docker
...
Installed Packages
Name        : docker
Arch        : x86_64
Version     : 1.6.0
Release     : 11.0.1.el7.centos
Size        : 32 M
Repo        : installed
From repo   : extras

$ yum info firewalld
...
Installed Packages
Name        : firewalld
Arch        : noarch
Version     : 0.3.9
Release     : 11.el7
Size        : 2.3 M
Repo        : installed
From repo   : anaconda

$ uname -r
3.10.0-229.4.2.el7.x86_64

$ cat /etc/redhat-release
CentOS Linux release 7.1.1503 (Core)
```

**Comment by jpopelka** [comment]
> Another app is currently holding the xtables lock. Perhaps you want to use the -w option?

It means that both docker and firewalld called iptables command at the same time.
This should be fixed with firewalld-0.3.13 (https://github.com/t-woerner/firewalld/commit/b3b451d6f8946986b8f50c8bcddeef50ed7a5f8f) or docker-1.7 (https://github.com/docker/docker/commit/8301dcc6d702a97feeb968ee79ae381fd8a4997a).

**Comment by jpopelka** [comment]
> Installed Packages
> Name : firewalld
> Version : 0.3.9-11.el7

It (https://bugzilla.redhat.com/show_bug.cgi?id=1161745) will most likely be fixed in RHEL-7.2

**Comment by thaJeztah** [comment]
Given the information that @jpopelka provided, I don't think it's necessary to keep this issue open, because it should be resolved in 1.7

If you think I closed this by mistake, feel free to comment here after me closing it.
